pkg/master/master_test.go

```diff
@@ -36,18 +36,7 @@ func TestGetServersToValidate(t *testing.T) {
 
 	master.nodeRegistry = registrytest.NewMinionRegistry([]string{"node1", "node2"}, api.NodeResources{})
 
-	servers := master.getServersToValidate(&config, true)
-
-	if len(servers) != 7 {
-		t.Errorf("unexpected server list: %#v", servers)
-	}
-	for _, server := range []string{"scheduler", "controller-manager", "etcd-0", "etcd-1", "etcd-2", "node-0", "node-1"} {
```

Comment:
Yes, that is what the code below does (out of frame of the diff). This first chunk validates that the getServersToValidate()flow works with nodes included, which we will no longer support.